src/gpu/intel/jit/ngen/ngen.hpp

```diff
@@ -1387,6 +1387,7 @@ template <typename... Targs> void ignoredep(Targs&&... args) { NGEN_NAMESPACE::B
 template <typename... Targs> void subdep(Targs&&... args) { NGEN_NAMESPACE::BinaryCodeGenerator<hw>::subdep(std::forward<Targs>(args)...); } \
 template <typename... Targs> void wrdep(Targs&&... args) { NGEN_NAMESPACE::BinaryCodeGenerator<hw>::wrdep(std::forward<Targs>(args)...); } \
 template <typename DT = void, typename... Targs> void min_(Targs&&... args) { NGEN_NAMESPACE::BinaryCodeGenerator<hw>::template min_<DT>(std::forward<Targs>(args)...); } \
+template <typename DT = void, typename... Targs> void amax_(Targs&&... args) { NGEN_NAMESPACE::BinaryCodeGenerator<hw>::template amax_<DT>(std::forward<Targs>(args)...); } \
```

Comment:
Thanks for the updates! Can you revert the remaining nGEN changes as well?

Reply:
Yes. Thanks.